### PR TITLE
Another revision

### PR DESCRIPTION
**Pull request by ypriverol**
@timosachsenberg @lgatto @Leprevost @ruiwanguk @julianu 

Hi all:

As I promise the present version contains all the major changes that where proposed by the reviewers except the major comments from Titus. 

Major changes:
- A new Supplementary Note was included to describe the details of some of the sections. 
- Examples were added to some of the rules as requested by one of the reviewers. **would be important if you add more examples**.
- A new table was added to the manuscript including some tutorials, courses, workshops, etc. This was 
  request by Reviewer 1 (Titus) in the first revision. 
- I added some new references to the manuscript related with the examples and missing citations that were recommend by @lgatto in issue #49 .

### CODE REVIEW

**Comment by ypriverol** [comment]
@timosachsenberg @lgatto  Can you accept this version.

**Comment by ypriverol** [comment]
@lgatto @timosachsenberg can anyone accept this pull-request.

File: REVIEWS.md [hunk 1]
```diff
@@ -52,12 +54,18 @@ to swallow.)
...
+and we highlighted the concern about the charge if the users exceed certain quotes.
+
 In the conclusions, the (substantial) learning curve of git is
 mentioned. Perhaps some pointers to open lessons should be made
 available, along with workshops? For example, both Data Carpentry and
 Software Carpentry teach git and github on a global basis.
 (Disclaimer: I am heavily involved with both.)
 
+> R/ We agree with the reviewer and in the present version we added some useful links to 
+documentation, coursers and tutorials.
```
Comment (timosachsenberg) [inline REVIEWS.md]: courses

File: REVIEWS.md [hunk 2]
```diff
@@ -67,8 +75,14 @@ Gitter is not owned by github, AFAIK. They just integrate. (Line 203).
 Why prioritize it over others? For example, Slack is another service
 that provides similar functionality. (We use gitter, but still.)
 
+> R/ We agree with the reviewer. In the present version we refined the paragraph making clear 
+that Gitter is not only chat system integrated with Github. We also included Gitter as an example of
```
Comment (timosachsenberg) [inline REVIEWS.md]: the only

File: REVIEWS.md [hunk 3]
```diff
@@ -67,8 +75,14 @@ Gitter is not owned by github, AFAIK. They just integrate. (Line 203).
 Why prioritize it over others? For example, Slack is another service
 that provides similar functionality. (We use gitter, but still.)
 
+> R/ We agree with the reviewer. In the present version we refined the paragraph making clear 
+that Gitter is not only chat system integrated with Github. We also included Gitter as an example of 
+chat system but leaving the space open for other systems.
```
Comment (timosachsenberg) [inline REVIEWS.md]: maybe rephrase: Gitter serves, amongst several alternatives, as a popular example for a github integrated chat system.

File: REVIEWS.md [hunk 4]
```diff
@@ -136,6 +150,9 @@ other version control systems. In the current version we removed the term from *
    commentary, and appropriate resolution. Similarly oriented links
    would help to convey the value / ease of continuous integration,
    and of test coverage.
+   
+>R/ We agree with the reviewer. In the present version of the manuscript we included examples
```
Comment (timosachsenberg) [inline REVIEWS.md:154]: the answer might need to be extended and state that we address both concerns

File: REVIEWS.md [hunk 5]
```diff
@@ -164,6 +185,9 @@ other version control systems. In the current version we removed the term from *
    as advertised. The second is the practice of committing (to master)
    running code. Maybe it would be better to break these out, and go
    for the novelty of 11, rather than the familiar 10, rules.
+   
+> We refined the **Rule 5** and it only provides information about unit test 
+and the idea of continues integration.
```
Comment (timosachsenberg) [inline REVIEWS.md]: continuous

File: REVIEWS.md [hunk 6]
```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
    think attempts to 'automatically update the documentation on code
    modification' are almost counter-productive -- availability of this
    level of documentation is not really what users crave.
+   
+> R/ We kept the current version of **Rule 5** about continues integration. The idea of the manuscript is to show
```
Comment (timosachsenberg) [inline REVIEWS.md]: continuous

File: REVIEWS.md [hunk 7]
```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
    think attempts to 'automatically update the documentation on code
    modification' are almost counter-productive -- availability of this
    level of documentation is not really what users crave.
+   
+> R/ We kept the current version of **Rule 5** about continues integration. The idea of the manuscript is to show 
+ the Git/GitHub infrastructure to bioinformatics community and general good practices. In general we don't aim to explain
```
Comment (timosachsenberg) [inline REVIEWS.md]: to the bioinformatics community

File: REVIEWS.md [hunk 8]
```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
...
+ in details best software engineering practices. However, **Rule 6** was extended commented in the supplmentary note explaining concepts like unit tests,
```
Comment (timosachsenberg) [inline REVIEWS.md]: In general, we don't aim to provide detailed explanations of best software engineering practices.

Comment (timosachsenberg) [inline REVIEWS.md]: - remove commented

File: REVIEWS.md [hunk 9]
```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
    think attempts to 'automatically update the documentation on code
    modification' are almost counter-productive -- availability of this
    level of documentation is not really what users crave.
+   
+> R/ We kept the current version of **Rule 5** about continues integration. The idea of the manuscript is to show 
+ the Git/GitHub infrastructure to bioinformatics community and general good practices. In general we don't aim to explain
+ in details best software engineering practices. However, **Rule 6** was extended commented in the supplmentary note explaining concepts like unit tests, 
+ and best practices of documentation. It should be notice by the reviewer that the current manuscript is about the use of Git/GitHub
```
Comment (timosachsenberg) [inline REVIEWS.md]: noticed

File: REVIEWS.md [hunk 10]
```diff
@@ -181,6 +205,13 @@ other version control systems. In the current version we removed the term from *
...
+ to improve the productivity in your research lab and team and not about general best practices of codding, documentation, etc.
```
Comment (timosachsenberg) [inline REVIEWS.md]: coding

File: REVIEWS.md (hunk 9 above)
Comment (timosachsenberg) [inline REVIEWS.md]: about the use of ?

File: document/Supplementary-Note.md [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+
+(6) Institute of Plant Biology and Biotechnology, University of Muenster, Schlossplatz 8, 48143 Muenster, Germany.
+
+[^1]: yperez@ebi.ac.uk.
+[^2]: juan@ebi.ac.uk.
+
+
+## Section 1: Git LSF file system
+
+Git Large File Storage (LFS) replaces large files (> 50 Mb)such as audio samples, videos, datasets, and graphics with text pointers inside Git, while
```
Comment (timosachsenberg) [inline document/Supplementary-Note.md:23]: space after )

File: document/Supplementary-Note.md [hunk 12]
```diff
@@ -0,0 +1,232 @@
...
+  git commit -m "Add design file"
+  
+  git push origin master
+ 
+``` 
+
+
+One of the major concerns about the use of GitHub is that some of the services are limited and can be charged if the users exceed certain quotes. For example, the Git LSF filesystem is available for all type of projects: personal repositories or organization accounts. A minimal space is provided for personal/organization repositories without charge
+(1GB). If you exceed this quota, you can still clone repositories with large assets, but you will only retrieve the pointer files, and you will not be able to push new files back up.
+
```
Comment (timosachsenberg) [inline document/Supplementary-Note.md:58]: Note that the use of GitHub LFS service is not mandatory and alternative providers, including self-hosting of files is possible.

File: REVIEWS.md (hunk 1 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 2 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 3 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 4 above)
Comment (ypriverol) [inline REVIEWS.md:154]: I think a simple sentence like this is enough.

File: REVIEWS.md (hunk 5 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 6 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 7 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

File: REVIEWS.md (hunk 8 above)
Comment (ypriverol) [inline REVIEWS.md]: Removed +1

File: REVIEWS.md (hunk 10 above)
Comment (ypriverol) [inline REVIEWS.md]: Done +1

**Comment by ypriverol** [comment]
@timosachsenberg I already included all your comments.

**Comment by lgatto** [comment]
Sorry for the silence these last days - have some important things to finish up asap - will catch up later.

**Comment by ypriverol** [comment]
Great hope @timosachsenberg will accept my latest changes and then we can move forward. I would lie to resubmit next week.

**Comment by lgatto** [comment]
Could we ask for comment from the community before resubmitting - Titus offered to disseminate the request.

**Comment by ypriverol** [comment]
Yes, in the current version the remaining points are the Titus comments. As we agree before, we should try to invite him an has a discussion (open) with him around these points and then we can arrive to the final solution.

**Comment by ypriverol** [comment]
@timosachsenberg if you have some time take a look to the current pull-request, I would like to move this forward

**Comment by ypriverol** [comment]
@lgatto @timosachsenberg @julianu @Leprevost @flu I can accept the pull-request.

**Comment by prvst** [comment]
@ypriverol  looks ok, want me to accept it?

**Comment by ypriverol** [comment]
Yes sure
